test/grnstate-tests.js

```diff
@@ -1,22 +1,23 @@
-// var assert = require("chai").assert;
-// import { grnState } from "./grnstate";
-
-/* TODO: Framework for front-end test
-
-var testResetNormalization = (input, propertyExpected) => {
-    assert.equal(propertyExpected, grnState.propertyValue);
-}
+/* eslint-disable max-len */
+/* eslint no-unused-vars: [2, {"varsIgnorePattern": "should"}] */
```

Comment:
Try to minimize these disables. Let me guess to see what you were trying to skip...